runtimes/pythonrt/py.go

```diff
@@ -63,7 +63,27 @@ func parsePyVersion(s string) (major, minor int, err error) {
 	return
 }
 
+func isFile(path string) bool {
+	info, err := os.Stat(path)
+	if err != nil {
+		return false
+	}
+
+	return !info.IsDir()
+}
+
+const exeEnvKey = "AK_PYTHON"
```

Comment:
I'd give it a more hierarchical name like AK__WORKERS__PYTHON_PATH
since this env var is irrelevant to AK itself, it is only relevant to ak's worker and specifically to python worker

Reply:
Changed to `AK_WORKER_PYTHON` (why the double `_`?)

Reply:
We have a convention we use __ to split env hierarchy 
This is for koanf to parse configurations correctly

I know you don't use koanf here, but we should probably keep the consistency